example/spec/parallel/dbRecords/DbRecordsDLQSpec.js

```diff
@@ -29,7 +29,7 @@ describe('When a bad record is sent on the DLQ', () => {
 
     const granuleId = randomString(10);
     const files = [fakeFileFactory()];
-    const granule = fakeGranuleFactoryV2({ files, granuleId });
+    const granule = fakeGranuleFactoryV2({ files, granuleId, published: false });
```

Comment:
This change generally makes sense, but what specifically required it?

Reply:
Attempting to delete a published granule results in an error https://github.com/nasa/cumulus/blob/master/packages/api/endpoints/granules.js#L207

Reply:
Ah, right, it's an endpoint constraint based on the key, not an actual CMR value.    👍🏻  Thanks for the clarification.